manifests/g/glzr-io/zebar/1.3.0/glzr-io.zebar.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: glzr-io.zebar
+PackageVersion: 1.3.0

```

```suggestion
PackageVersion: 0.0.0
```
Please update path to match this change.